extensions/notion/package.json

```
@@ -29,7 +29,17 @@
       "title": "Create Database Page",
       "subtitle": "Notion",
       "description": "Create a page in a Notion database.",
-      "mode": "view"
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "closeAfterCreate",
+          "type": "checkbox",
+          "label": "Close raycast after create page",
+          "required": false,
+          "default": true,
```

```suggestion
          "default": false,
```

Let's disable it by default so that it doesn't disrupt current users' workflows.